providers/jiva-csi-provisioner/test.yml

```diff
@@ -0,0 +1,118 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+          ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+          ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+
+        - name: Install Jiva operator in operator namespace
+          shell: |
+            kubectl apply -f https://raw.githubusercontent.com/openebs/jiva-operator/master/deploy/crds/openebs_v1alpha1_jivavolume_crd.yaml
+            kubectl apply -f https://raw.githubusercontent.com/openebs/jiva-operator/master/deploy/service_account.yaml
+            kubectl apply -f https://raw.githubusercontent.com/openebs/jiva-operator/master/deploy/role.yaml
+            kubectl apply -f https://raw.githubusercontent.com/openebs/jiva-operator/master/deploy/role_binding.yaml
+            kubectl apply -f https://raw.githubusercontent.com/openebs/jiva-operator/master/deploy/operator.yaml
```

Comment:
in jiva operator, the images has to be updated.

Reply:
done